package/src/main.rs

```diff
@@ -131,10 +131,7 @@ macro_rules! build_step {
 
 macro_rules! integration_test {
     ($msg:expr $(,)?) => {
-        log!(Color::Magenta, ">> {}", $msg);
-    };
-    ($msg:expr, $($arg:tt)*) => {
-        log!(Color::Magenta, ">> {}", format!($msg, $($arg)*));
+        log!(Color::Magenta, ">> {}", format!($msg));
```

Comment:
You should be able to drop the format! when you circle back and just pass $msg directly.

Reply:
I'm not so sure, as the reason I touched this part was due to (from my console output):
```
>> Verify usage of a Pants {pants_release} on the pants repo.
```

so the arg-less version of the macro needs the `format!` it seems.

Reply:
and I dropped the arg version as after the Rust upgrade, it seems embedding the args are the new way to go about formatting.. :)

Reply:
Yep, confirmed. Still need `format!`.